distribution/bin/shutdown.sh

```diff
@@ -15,7 +15,7 @@
 cd `dirname $0`/../target
 target_dir=`pwd`
 
-pid=`ps ax | grep -i 'nacos.nacos' | grep ${target_dir} | grep java | grep -v grep | awk '{print $1}'`
+pid=`ps ax | grep -i 'nacos.nacos' | grep -i ${target_dir} | grep java | grep -v grep | awk '{print $1}'`
```

Comment:
Why ignore case of nacos dir? Can you open an issue to describe the situation first?